Ablauf.md

```diff
@@ -42,7 +42,7 @@ Alle machen was, wissen welche Rolle sie haben. ca. 2:30h
 Abschlusspräsentationen
 -----------------------
 
-Ca. eine halbe Stunde vor Ende sollte angesagt werden, dass in einer halben Stunde Presentationen losgehen. Dabei sollte auch schon erwähnt werden, worum es da geht.
+Ca. eine halbe Stunde vor Ende sollte angesagt werden, dass in einer virtel Stunde Presentationen losgehen und dannach ende ist. Dabei sollte auch schon erwähnt werden, worum es da geht.
```

Comment:
> viertel

> Ende

Reply:
Die Präsentationen sind doch immer 16:45 weil 17:00 viele gehen müssen

Reply:
Das stimmt. So sind die Worte in der deutschen Rechtschreibung: "viertel" (groß oder klein weiß ich nicht) und "Ende" und "danach". Das ist, was ich sagen wollte. Kein Zweifel am Inhalt :+1: